tests/integration/scripts/general

```diff
@@ -25,6 +25,8 @@ args = parser.parse_args()
 
 if args.allocate is not None:
     data = b"x" * args.allocate
+    # force interpreter to actually allocate
+    cnt = len(data)
```

Comment:
I am pretty sure that `b"x" * n` allocates immediately.  If you see otherwise, could you please provide a recipe that demonstrates this?

Reply:
I could swear that it was failing reliably before the change, but can no longer reproduce. Reverted.